src/Bijectors.jl

```diff
@@ -244,6 +284,58 @@ function invlink(
     return x
 end
 
+function invlink_jacobian(
+    d::SimplexDistribution, 
+    y::AbstractVector{T}, 
+    ::Type{Val{proj}} = Val{true}
+) where {T<:Real, proj}
+    x, K = similar(y), length(y)
+    dxdy = similar(y, length(y), length(y))
+    @inbounds dxdy .= 0
+
+    ϵ = _eps(T)
+    @inbounds z = StatsFuns.logistic(y[1] - log(T(K - 1)))
+    unclamped_x = (z - ϵ) / (one(T) - 2ϵ)
+    @inbounds x[1] = _clamp(unclamped_x, d)
+    @inbounds if unclamped_x == x[1]
+        dxdy[1,1] = z * (1 - z) / (one(T) - 2ϵ)
+    end
+    sum_tmp = zero(T)
+    @inbounds for k = 2:(K - 1)
+        z = StatsFuns.logistic(y[k] - log(T(K - k)))
+        sum_tmp += x[k-1]
+        unclamped_x = ((one(T) + ϵ) - sum_tmp) / (one(T) - 2ϵ) * z - ϵ
+        x[k] = _clamp(unclamped_x, d)
+        if unclamped_x == x[k]
+            dxdy[k,k] = z * (1 - z) * ((one(T) + ϵ) - sum_tmp) / (one(T) - 2ϵ)
+            for i in 1:k-1
+                for j in i:k-1
+                    dxdy[k,i] += -dxdy[j,i] * z / (one(T) - 2ϵ)
+                end
+            end
+        end
+    end
+    @inbounds sum_tmp += x[K - 1]
+    @inbounds if proj
+    	unclamped_x = one(T) - sum_tmp
+        x[K] = _clamp(unclamped_x, d)
+    else
+    	unclamped_x = one(T) - sum_tmp - y[K]
+        x[K] = _clamp(unclamped_x, d)
+        if unclamped_x == x[K]
+            dxdy[K,K] = -1
+        end
+    end
+    @inbounds if unclamped_x == x[K]
+        for i in 1:K-1
+            @simd for j in i:K-1
+                dxdy[K,i] += -dxdy[j,i]
+            end
+        end
+    end
+    return LowerTriangular(dxdy)
```

Comment:
My own comments:
```
    # lower-triangular because `sum_tmp` for `k` only depends on `yᵢ` for `i = 1:(k - 1)`
    return LowerTriangular(dxdy)
```